src/test/common/CMakeLists.txt

```diff
@@ -453,42 +453,6 @@ if(WITH_SYSTEMD)
   add_ceph_unittest(unittest_journald_logger)
 endif()
 
-# Validation of the DECODE_START's struct_v compile-time checker.
-# First, ensure buildability of the test program itself. This is
-# useful to avoid false positives coming from other-than-the-assert
-# build failures.
-add_executable(unittest_decode_start_v_checker
```

Comment:
Nit: Can we comment them out and link to the [tracker](https://tracker.ceph.com/issues/70700#change-291874) if disabling the tests is temporary? If we plan to remove them permanently, it's fine as is.